### PR TITLE
Table: add summary row, fixed #1307, fixed #4451

### LINKED ISSUE #1307: [Table] 希望增加汇总的功能
table 中能否增加一个汇总的功能，能够自动汇总当前需要汇总的字段，或者可以定义显示格式

### LINKED ISSUE #4451: [Bug Report] el-table's sortable why not synchronization
### Element UI version
1.2.9

### OS/Browsers version
win7/firefox

### Vue version
2.2.6

### Reproduction Link
[https://jsfiddle.net/mq5vhpmy/](https://jsfiddle.net/mq5vhpmy/)

### Steps to reproduce
Switch the sortable value

### What is Expected?
sortable can realize the sort synchronization

### What is actually happening?
can't sync sorting

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

测试还没写，等发完 beta 再写。

### CODE REVIEW

**Comment by liril-net** [comment]
所以，等写完测试再合嘛

**Comment by Leopoldthecoder** [comment]
@liril-net 先合再写测试🌚

File: packages/table/src/table-body.js [hunk 1]
```diff
@@ -78,6 +78,8 @@ export default {
                 : ''
               ]
             ).concat(
+              this._self.$parent.$slots.append ? this._self.$parent.$slots.append : ''
```
Comment (QingWei-Li) [inline packages/table/src/table-body.js]: `this._self.$parent.$slots.append` 不就完了么为啥还要判断

Comment (Leopoldthecoder) [inline packages/table/src/table-body.js]: 说好不看的呢！

**Pull request merged**